build/config.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Copyright © 2018 Lisk Foundation
+#
+# See the LICENSE file at the top-level directory of this distribution
+# for licensing information.
+#
+# Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+# no part of this software, including this file, may be copied, modified,
+# propagated, or distributed except according to the terms contained in the
+# LICENSE file.
+#
+# Removal or modification of this copyright notice is prohibited.
+
+# shellcheck disable=SC2034
+
+VERSION="$VERSION"
+OS=$(uname)
+ARCH="${ARCH:-$( uname -m )}"
+BUILD_NAME="lisk-$VERSION-$OS-$ARCH"
+TARGET=""
+
+ncpu=$( grep -c processor /proc/cpuinfo )
+ncpu=$(( ncpu + 0 ))
+tcpu=$(( ncpu / 2 + 1 ))
+[[ $tcpu -gt 8 ]] && tcpu=8
+MAKEOPTS="-j${tcpu}"
+
+LISK_DIR="$VERSION"
+LISK_FILE="lisk-$VERSION.tgz"
+LISK_NETWORK="$LISK_NETWORK"
+LISK_URL="https://downloads.lisk.io/lisk/$LISK_NETWORK/$VERSION/$LISK_FILE"
+
+LISK_SCRIPTS_VERSION="0.4.0"
+LISK_SCRIPTS_SHA256SUM="03ddb4d651bba12906c77a6f1c3a6931c0e227bcd20cf44d409c0cd59c8d7b12"
+LISK_SCRIPTS_DIR="lisk-scripts-$LISK_SCRIPTS_VERSION"
+LISK_SCRIPTS_FILE="$LISK_SCRIPTS_DIR.tar.gz"
+LISK_SCRIPTS_URL="https://github.com/LiskHQ/lisk-scripts/archive/v$LISK_SCRIPTS_VERSION.tar.gz"
+
+NODE_VERSION="8.13.0"
```

Comment:
I will create a PR to update the node version to 8.13.0 for the sources as well.